airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -47,7 +47,7 @@
 # the initial values should be saved once and tracked for each stream, inclusively.
 GoogleAnalyticsQuotaHandler: GoogleAnalyticsApiQuota = GoogleAnalyticsApiQuota()
 
-LOOKBACK_WINDOW = datetime.timedelta(days=2)
+LOOKBACK_WINDOW = 2
```

Comment:
Maybe rename this to `DEFAULT_LOOKBACK_WINDOW`

Reply:
https://github.com/airbytehq/airbyte/pull/38884/commits/9d90f2c2897ef21d332f646fd3fb735a08b34cb0